docs/reference/rest-api/logs.asciidoc

```diff
@@ -0,0 +1,278 @@
+
+////
+[source,console]
+----
+POST logs-myapp-default/_refresh
+----
+// TEST[continued]
+// commented out to avoid documenting that a _refresh will always be sufficient to make the logs searchable
+// in the future, logs may be buffered and asynchronously processed
+////
+
+[source,console]
+------------------------------------------------------------
+POST logs-myapp-default/_search?q=custom:value
+------------------------------------------------------------
+// TEST[continued]
+
+The API returns the following response:
+
+[source,console-result]
+----
+{
+  "took": 5,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 1.0,
+    "hits": [
+      {
+        "_index": ".ds-logs-foo-default-2016.05.23-000001",
+        "_id": "FKgQT4IBWsM7OYMsIp0N",
+        "_score": 1.0,
+        "_source": {
+          "@timestamp": 1463990734853,
+          "message": "Hello",
+          "custom": "value",
+          "service": {
+            "name": "myapp"
+          },
+          "data_stream": {
+            "type": "logs",
+            "dataset": "myapp",
+            "namespace": "default"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/"took": 5/"took": $body.took/]
+// TESTRESPONSE[s/"_index": ".*"/"_index": $body.hits.hits.0._index/]
+// TESTRESPONSE[s/"_id": ".*"/"_id": $body.hits.hits.0._id/]
+// TESTRESPONSE[s/"_source": \{\n/"_source": \{\n"error_trace": "true",\n/]
+// The test system adds an error_trace:true parameter to all requests,
+// including the logs API which interprets it as global metadata that's added to every event
+
+'''
+
+Ingests a single log into the `logs-myapp-default` data stream.
+
+[source,console]
+------------------------------------------------------------
+POST _logs/myapp
+{"_metadata": {"service.name": "myapp"}} <1>
+{"@timestamp": "2016-05-23T08:05:34.853Z", "message": "Hello World"} <2>
+------------------------------------------------------------
+
+<1> Provides global metadata that applies to all log events in the request via a global metadata event.
+<2> Specifies the timestamp as an ISO date string.
+
+'''
+
+Ingests a two log events into the `logs-myapp-default` and `logs-my_other_app-default` data stream, respectively.
+Provides metadata via local metadata events.
+
+[source,console]
+------------------------------------------------------------
+POST _logs
+{"_metadata": {"service.name": "myapp"}} <1>
+{"_metadata": {"data_stream.dataset": "myapp"}} <2>
+{"@timestamp": "2016-05-23T08:05:34.853Z", "message": "hello app"} <3>
+{"_metadata": {"data_stream.dataset": "my_other_app"}} <4>
+{"@timestamp": "2016-05-23T08:05:34.853Z", "message": "other app"} <5>
+------------------------------------------------------------
+
+<1> Provides global metadata that applies to all log events in the request via a global metadata event.
```

Comment:
I understand the logic but not sure if this is intuitive. Maybe instead call it `_meta_global` and `_meta_local` or similar to make it more describtive

Reply:
Fair point